docs/user-guide/ffi.md

```diff
@@ -0,0 +1,122 @@
+# PyScript FFI
+
+The reason we decided to incrementally provide a unified *pyscript.ffi* utility is that [Pyodide](https://pyodide.org/en/stable/usage/api/python-api/ffi.html)'s *ffi* is only partially implemented in *MicroPython* but there are fundamentals differences even with the few common utilities both projects provide, hence our intention to smooth out their usage with all the features or caveats one needs to be aware of.
+
+Our `pyscript.ffi` offers the following utilities:
+
+  * `ffi.to_js(reference)` to convert any Python reference to its *JS* counterpart
+  * `ffi.create_proxy(def_or_lambda)` to proxy a generic Python's function into a *JS* one, without destroying its reference right away
+
+## to_js
+
+In [Pyodide project](https://pyodide.org/en/stable/usage/api/python-api/ffi.html#pyodide.ffi.to_js), this utility converts by default Python dictionaries into a *Map*, or prefer the *Map* indirection as the *Map* somehow better reflects the `obj.get(field)` native Python way to retrieve a value from a dictionary.
```

Comment:
* "In Pyodide project" -> "In the Pyodide project"
* "this utility converts by default" -> drop "by default"
* ", or prefer the *Map* indirection as the *Map* somehow better reflects" -> ". the *Map* object more accurately reflects"